### ISSUE TITLE
Add an option to replace every occurrence of e typo in the PR

### ISSUE THREAD

**Issue by alexisjanvier** [marmelab/sedy#37]
For exemple, replace all `develoment` occurrences by `development` in a single comment.
`s/develoment/development/ -g` ?

**Comment by Kmaschta** [comment]
Absolutely. Include sed flags (at least global `g` and case insensitive `i`) would be great.

**Comment by Kmaschta** [comment]
This issue depend on #34
